lms/djangoapps/learner_dashboard/tests/test_utils.py

```diff
@@ -3,9 +3,16 @@
 """
 
 import ddt
-from django.test import TestCase
+
+from datetime import date
+
+from django.test import TestCase, override_settings
+from django.contrib.auth.models import User
+
+from student.models import UserProfile
 
 from lms.djangoapps.learner_dashboard import utils
+from lms.djangoapps.learner_dashboard.utils import disclaimer_incomplete_fields_notification
```

Comment:
`from lms.djangoapps.learner_dashboard import utils` is above this line. You don't have to import this explicitly -- just use `utils.disclaimer_incomplete_fields_notification`.